index.html

```diff
@@ -1,132 +1,89 @@
+                </p>
+                <p>
+                   Prior to joining industry, I was pursuing my Ph.D. in Computer Science but had a change of heart and opted for an M.S. instead. While in academia, I worked in a number of research areas such as biometrics, bioinformatics, and mobile device security. My graduate thesis focused on visualizing genomic data in the browser.
+                </p>
+                <p>
+                    Please see my <a href="phillip.whisenhunt.resume.pdf" target="_blank">resume</a> linked above for a more complete picture of my work history.
+                </p>
             </div>
-            <footer class="row">
-                <div class="col-md-2"></div>
-                <div class="col-md-8">
-                    <p>
-                        <a href="https://github.com/pwhisenhunt/pwhisenhunt.github.com" alt="Phillip Whisenhunt Website Repository">
-                        </a>This website was built using awesome open source projects <a href="http://getbootstrap.com/" alt="Twitter Bootstrap">Twitter Bootstrap</a> and <a href="http://html5boilerplate.com/" alt="HTML5 Boilerplate">HTML5 Boilerplate</a>.
-                        This website does not reflect the opinions of my past or present employers.
-                    </p>
-                </div>
-                <div class="col-md-2"></div>
+        </div>
+         <div class="information-container">
+            <h2>Life<span class="heading-decoration"></span></h2>
+            <div class="information-text">
+                <ul class="emojis">
+                    <list class="emoji">🏃‍♂️</list>
+                    <list class="emoji">🧘</list>
+                    <list class="emoji">👨‍🍳</list>
+                    <list class="emoji">📗</list>
+                    <list class="emoji">🎮</list>
+                    <list class="emoji">🛩️</list>
+                    <list class="emoji">☀️</list>
+                    <list class="emoji">🏝️</list>
+                </ul>
             </div>
         </div>
-        <script>
-            /* Google analytics */
-            var _gaq=[['_setAccount', 'UA-25658199-1'],['_trackPageview']];
-            (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
-            g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
-            s.parentNode.insertBefore(g,s)}(document,'script'));
-        </script>
-    </body>
-</html>
+    </section>
+    <section id="writing-section" class="writing-section">
+        <h2>Check Out My <span class="heading-emphasis">Writing</span><span class="heading-decoration"></span></h2>
```

Comment:
You could do an `:after` for the h2's instead of doing spans but spans are also ok